libs/community/langchain_community/document_loaders/parsers/audio.py

```diff
@@ -12,14 +13,273 @@
 logger = logging.getLogger(__name__)
 
 
+class AzureOpenAIWhisperParser(BaseBlobParser):
+    """
+    Transcribe and parse audio files using Azure OpenAI Whisper.
+
+    This parser integrates with the Azure OpenAI Whisper model to transcribe
+    audio files. It differs from the standard OpenAI Whisper parser, requiring
+    an Azure API key and endpoint. The parser is limited to files under 25 MB.
+
+    **Note**:
+    While this parser uses the OpenAI API, it is configured for Azure OpenAI
+    services. The key difference is the integration with the Azure ecosystem,
+    making it suitable for workflows involving other Azure services.
+
+    For files larger than 25 MB, consider using Azure AI Speech batch transcription:
+    https://learn.microsoft.com/azure/ai-services/speech-service/batch-transcription-create?pivots=rest-api#use-a-whisper-model
+
+    Setup:
+        1. Follow the instructions here to deploy Azure Whisper:
+           https://learn.microsoft.com/azure/ai-services/openai/whisper-quickstart?tabs=command-line%2Cpython-new&pivots=programming-language-python
+        2. Install ``langchain`` and set the following environment variables:
+
+        .. code-block:: bash
+
+            pip install -U langchain
```

Comment:
my bad, I thought it could be accessed just by installing LangChain, but after checking, that's not true, has been updated to:

```python
pip install -U langchain langchain-community
```